### PR TITLE
Endpoints to fetch adb logs using UDID

### PR DESCRIPTION
**Pull request by shridharkalagi**

### CODE REVIEW

**Comment by shridharkalagi** [comment]
Fixing issue #6

File: src/main/java/com/hariharanweb/remoteappiummanager/Server.java [hunk 1]
```diff
@@ -43,6 +43,14 @@ public static void main(String[] args) throws IOException {
             });
         });
 
+        path("/startadblog", () -> {
```
Comment (bsneha90) [inline src/main/java/com/hariharanweb/remoteappiummanager/Server.java]: Can we change the path as : '/devices/adblog/start' & '/device/adblog/stop/:udid' ?

Comment (shridharkalagi) [reply src/main/java/com/hariharanweb/remoteappiummanager/Server.java]: Ya I will update the paths

File: src/main/java/com/hariharanweb/remoteappiummanager/Server.java [hunk 2]
```diff
@@ -43,6 +43,14 @@ public static void main(String[] args) throws IOException {
             });
         });
 
+        path("/device/adblog/start", () -> {
```
Comment (SrinivasanTarget) [inline src/main/java/com/hariharanweb/remoteappiummanager/Server.java]: It should be move above inside existing `/device` [path](https://github.com/hariharanwebmail/RemoteAppiumManager/pull/22/files#diff-766c47a020090f03eda1fbd52c17dc0dL35) and create inner path like below,
```
path("/adblog", () -> {
                 get("/start", deviceController. startADBLog, new JsonTransformer());
                 get("/stop", deviceController.stopADBLog, new JsonTransformer());
              });
```

**Pull request merged**